README.md

```diff
@@ -50,6 +50,10 @@ The following workflow is for a new **Helm** operator:
 - Optional: [dep][dep_tool] version v0.5.0+.
 - Optional: [delve](https://github.com/go-delve/delve/tree/master/Documentation/installation) version 1.2.0+ (for `up local --enable-delve`).
 
+## Operator capability level
+
+![operator-capability-level](./doc/images/operator-capability-level.png)
+
```

Comment:
I think we should include this blurb above the image:

> Each operator type has a different set of capabilities. When choosing what type to use for your project, it is important to understand the features and limitations of each of the project types and the use cases for your operator.

WDYT of moving this into the `## User Guides` section as a subheader there since that's where the links to each of the user guides for Go, Ansible, and Helm are?

Reply:
Nope! Still in the README. There's a `## User Guides` section (around line 139) that links to the individual project type user guides. That seems like a natural place for this since that's the point users have to decide which user guide to follow.

It's like a [Choose your own adventure](https://en.wikipedia.org/wiki/Choose_Your_Own_Adventure) :)

Reply:
Found tks 👍

Reply:
HI, @joelanford it is done. Is this small change ok for you? Could we move forward here?